distributed/tests/test_executor.py

```diff
@@ -3576,3 +3576,25 @@ def test_idempotence(s, a, b):
 
     yield e._shutdown()
     yield f._shutdown()
+
+
+@gen_cluster(executor=True)
+def test_make_serializable(e, s, a, b):
+    from distributed.executor import serialize_for_publish, deserialize_published
+    import pickle
+    f = delayed(lambda: 1)()
+    futures = yield e._scatter([f, f, f])
+    with pytest.raises(AttributeError):
+        pickle.dumps(futures)
+    with serialize_for_publish(futures) as f:
+        d = pickle.dumps(f)
+    assert d
+
+
+@gen_cluster(executor=True)
+def test_publish_simple(e, s, a, b):
+    data = yield e._scatter(range(12))
+    e.publish_dataset(data, 'data')
+    yield gen.sleep(0.2)
+
+    assert s.published_data
```

Comment:
yeah, sorry, this comment was intended to be in the same space of the tests, not referring to this test in particular.  I ought to have been more clear from the get-go.

The `start=True` keyword will call `sync(loop, e._start)`, which is bad because we can't use any synchronous calls within the `@gen_cluster` tests.  I wouldn't worry about this for now.  You rarely need to care.  We only have to deal with this this time because we need multiple executors, and so are slightly outside of the abilities of `@gen_cluster`.